### PR TITLE
#112783759 Update EventX UI

### PR DESCRIPTION
**Pull request by andela-fsenjobi**
#### What does this PR do?
Updates the create event preview page to use a modal
Fixes footer issue. Make the footer run across the whole page
Makes side nav a collapsible nav
Makes category sidebar into filters
Improve the ticket report page appearance
Display highlight, review and sponsor headers only when there is content
Change the floating ticket delete button
Update the template selector to be more appealing
#### How should this be manually tested?
These changes can be seen on visiting the said pages
#### Any background context you want to provide?
The footer covered portions of the sidebar when the page has little or no content
Side nav was static. This reduced the portion of the page that was usable
##### New look of the Side Nav
image
##### New look of the footer and a portion of the dashboard
image
##### New look of the ticket reports page
image
##### New event preview page
image
##### New look of the template selector
image
##### New look of the ticket type form
image

### CODE REVIEW

**Comment by andela-gogbara** [comment]
Nice Job @andela-fsenjobi I love the new event preview UI.

File: app/assets/javascripts/bind_event_to_preview_page.js [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
-  // script for binding event details to preview page
-  // var prev_color = "init";
-  // var color = "";
-  var prev_color = '';
-  var color = '';
+
   $('.preview').click(function () {
-    // if(prev_color=="init" || prev_color!=color){
-    //   prev_color = color
     var start_date = $('#event_start_date').val();
```
Comment (andela-sdamian) [inline app/assets/javascripts/bind_event_to_preview_page.js]: use single `var`

File: app/assets/javascripts/bind_event_to_preview_page.js [hunk 2]
```diff
@@ -1,24 +1,20 @@
 $(document).ready(function () {
-  var event_date = $('.parallax-container').data('countdown');
+  var event_date = $('.parallax-container').data('countdown'),
+      prev_color = '',
+      color = '',
+      map;
+
   if (event_date) {
```
Comment (andela-sdamian) [inline app/assets/javascripts/bind_event_to_preview_page.js]: place body of if statement on a single line if not more than one

**Comment by andela-oesho** [comment]
👍  Excellent Job Femi, I still think the ticket report page can be improved on like using colours and cards rather than table, I also love the new template selector.

**Pull request merged**